daemon/openrazer_daemon/hardware/mouse.py

```diff
@@ -11,6 +11,74 @@
     get_backlight_active as _da_get_backlight_active
 from openrazer_daemon.dbus_services.dbus_methods.chroma_keyboard import get_brightness as _get_backlight_brightness, set_brightness as _set_backlight_brightness
 from openrazer_daemon.misc.key_event_management import NagaHexV2KeyManager as _NagaHexV2KeyManager
+from openrazer_daemon.dbus_services.dbus_methods.lanceheadte import get_left_brightness as _get_left_brightness, get_right_brightness as _get_right_brightness, \
+    set_left_brightness as _set_left_brightness, set_right_brightness as _set_right_brightness
+
+
+class RazerLanceheadTE(__RazerDeviceSpecialBrightnessSuspend):
+    """
+    Class for the Razer Lancehead Tournament Edition
+    """
+    EVENT_FILE_REGEX = re.compile(r'.*Razer_Lancehead_TE-if0(1|2)-event-kbd')
+
+    USB_VID = 0x1532
+    USB_PID = 0x0060
+    HAS_MATRIX = True
+    WAVE_DIRS = (1, 2)
+    MATRIX_DIMS = [1, 16]
```

Comment:
Can you try `[8, 6]` here and check if this works: ![](https://developer.razer.com/wp-content/uploads/led-profile-super-mouse-grid-org.png)

Reply:
What??? Why would this be the case? Let me explain.
The mouse accepts a single command to set the colors for all LEDs.
This "special matrix" designed to fit all devices isn't sent verbatim, but rather mapped per device in the Chroma service to make it easier for developers to target all devices easily.
OpenRazer simply provides the "single command" 17x1 matrix, and changing this would require rewriting a huge part of the daemon.

Reply:
Please read my message:

> and check if this works: 

**check if this works**

Reply:
It wouldn't work, guaranteed.

Reply:
@z3ntu Only 0,0 through 0,5 had effect on a portion of the LEDs when checking with the matrix discovery tool using the requested values.